src/software/logger/logger.h

```diff
@@ -30,8 +38,33 @@ class LoggerSingleton
     LoggerSingleton()
     {
         logWorker = g3::LogWorker::createLogWorker();
+        // Robot diagnostics logs are in
+        // bazel-out/k8-fastbuild/bin/software/gui/robot_diagnostics/robot_diagnostics_main.runfiles/__main__/software/
+        // Full system logs are in
+        // bazel-out/k8-fastbuild/bin/software/full_system.runfiles/__main__/software/
+        // Simulated test logs are in
+        // bazel-out/k8-fastbuild/bin/software/simulated_tests/TEST_NAME.runfiles/__main__/software/
+        // where TEST_NAME is the name of the simulated test
```

Comment:
can we re-route these to a gitignored folder? like `software/logs/robot_diagnostics`, `software/logs/full_system` and the contents of the logs folder would just be gitignored. (The logs folder itself will be staged so we don't need to write code to create that folder every time)

Reply:
I believe this would violate Bazel's hermetic build principles but ill double-check and get back to you

Reply:
so by default this isn't possible because the working directory is set to bazel-out but we can instead specify --logging_dir as a command line arg. .log files are already in the .gitignore